src/pages/patterns/filters.mdx

```diff
@@ -0,0 +1,416 @@
+| Many categories, live updates                            | A user wants to set multiple filters across a dataset with many categories and see their dataset update live       | Sheet                                    |
+| Many categories, batch updates                           | A user wants to set multiple filters across a dataset with many categories and apply them all at once              | Drawer                                   |
+
+## Usage
+
+### Basic filtering
+
+Basic Filters allow users to filter a dataset through one category with only a few options.
+
+#### Single select
+
+Single select basic filters use one selection to filter data. Upon the selection,
+the Menu closes and the filter is surfaced as a Tag.
+Selecting another menu option replaces the current Filter with the new selection.
+
+![Filter pattern basic single select 1](figma:filter-usage-basic-singleselect-1.png)
+![Filter pattern basic single select 2](figma:filter-usage-basic-singleselect-2.png)
+![Filter pattern basic single select 3](figma:filter-usage-basic-singleselect-3.png)
+![Filter pattern basic single select 4](figma:filter-usage-basic-singleselect-4.png)
+
+#### Multiple selections
+
+Multiple selection basic filters use multiple selections within a category to filter data.
+The Menu remains open as the user makes their selections.
+The selections are surfaced as Tags with a Basic Button to clear all filters.
+
+![Filter pattern basic multiple select 1](figma:filter-usage-basic-multipleselect-1.png)
+
+---
+
+![Filter pattern basic multiple select 2](figma:filter-usage-basic-multipleselect-2.png)
+
+#### Nested menu
+
+A [Nested menu](/components/menu) can be used to organize subcategories of data that can be used to filter a dataset.
+
+![Filter pattern basic nested menu 1](figma:filter-usage-basic-nestedmenu-1.png)
+
+---
+
+![Filter pattern basic nested menu 2](figma:filter-usage-basic-nestedmenu-2.png)
+
+---
+
+![Filter pattern basic nested menu 3](figma:filter-usage-basic-nestedmenu-3.png)
+
+### Advanced filtering
+
+Advanced Filters allow users to set and configure multiple filters on a dataset.
+
+#### Tooltip modal
+
+A Tooltip modal allows users to filter a dataset through one category with many options,
+and allows the user to search for specific options.
+They can also be part of a more complex flow.
+
+![Filter pattern advanced tooltip modal](figma:filter-usage-advanced-tooltip-modal.png)
+
+#### Sheet
+
+A Sheet is used when the dataset is filtered live as the user makes their selections.
+Selections are surfaced as a Tag as the user sets each filter.
+
+![Filter pattern advanced sheet 1](figma:filter-usage-advanced-sheet-1.png)
+
+---
+
+![Filter pattern advanced sheet 2](figma:filter-usage-advanced-sheet-2.png)
+
+---
+
+![Filter pattern advanced sheet 3](figma:filter-usage-advanced-sheet-3.png)
+
+---
+
+![Filter pattern advanced sheet 4](figma:filter-usage-advanced-sheet-4.png)
```

Comment:
Unfortunately, Garden's [DatepickerRange](https://garden.zendesk.com/components/date-picker#date-range) does not currently work as portrayed by this dialog. It would be good to get the graphic to correspond with Garden's supported component set.

Reply:
Removed DatepickerRange example from all visuals, should be reflected in the visuals now